src/components/RegistrationList.tsx

```diff
@@ -115,14 +115,26 @@ export const RegistrationListItemContent = ({
 
           {ticketView &&
             (registration.status === RegistrationStatus.Draft || registration.status === RegistrationStatus.New) && (
-              <Typography
-                sx={{
-                  p: '0.1rem 0.75rem',
-                  bgcolor: 'primary.light',
-                  color: 'primary.contrastText',
-                }}>
-                {t('registration.public_page.result_not_published')}
-              </Typography>
+              <Box sx={{ mb: '0.5rem', textAlign: 'center' }}>
+                <Typography
+                  sx={{
```

Comment:
Denne klarer jeg ikke å finne på sidene?

Reply:
Denne mener du? 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/f2cbad59-d257-49c3-9dd9-e6b6dda9d4d2)

Jeg hadde en annen oppfatning av hva "Add following tekst to blue box" betydde da, og trodde det var boksen i høyremeyen 🤔 Vet JEG har snakka om at vi trenger tekst som beskriver hva publisering/DOI/kuratorstøtte-boksene faktisk er. Men mulig jeg tar feil her altså

Reply:
LOL, jeg skal få flyttet den 🤣 
Syntes det var merkelig med masse tekst om at du måtte få hjelp av en kurator "ved din enhet på din institusjon" teksten uten noen som helst forklaring på hvordan du skulle få hjelp.

Reply:
Det er på oversikten over oppgaver, for å vise hvilke oppgaver som er knyttet til resultat som ikke enda er publiserte. Så vil tro den ikke skal vises der siden man allerede har opprettet en ticket for å kunne vises her 😅